src/linker/runner.py

```diff
@@ -8,24 +8,17 @@
 
 from linker.configuration import Config
 from linker.pipeline import Pipeline
-from linker.utilities.data_utils import copy_configuration_files_to_results_directory
 from linker.utilities.paths import LINKER_TEMP
 from linker.utilities.slurm_utils import is_on_slurm
 
 
-def main(
-    config: Config,
-    results_dir: Path,
-) -> None:
+def main(config: Config) -> None:
     """Set up and run the pipeline"""
 
     pipeline = Pipeline(config)
-
-    # Now that all validation is done, copy the configuration files to the results directory
-    copy_configuration_files_to_results_directory(config, results_dir)
```

Comment:
As mentioned elsewhere, I did this here b/c I wanted Pipeline() to validate as well before copying. Not a big deal though I don't think